### PR TITLE
Publish topic /foot_temperature

### PR DESCRIPTION
**Pull request by SFhmichael**
Using Kmeans to divide the image into 3 parts.
Find the median part and treat it as the temperature of food inside the wok, since the max part is the wok and the min part is background.
Publish topic named /food_temperature to represent the temperature of food.

### CODE REVIEW

File: src/OptrisColorconvert.cpp [hunk 1]
```diff
@@ -191,42 +194,38 @@ namespace optris_drivers2
...
-          if (k == maxTempIdx){
+          if (k == midTempIndex){
+            //if (1){
             for (const auto& contour : contours) {
               std::vector<cv::Point> resizedContour;
               for (size_t i = 0; i < contour.size(); i++) {
                   resizedContour.push_back(cv::Point(static_cast<int>(contour[i].x * scaleX), static_cast<int>(contour[i].y * scaleY)));
               }
-              cv::polylines(resizedImage, std::vector<std::vector<cv::Point>>{resizedContour}, true, cv::Scalar(0, 255, 0), 1);
+              cv::polylines(resizedImage, std::vector<std::vector<cv::Point>>{resizedContour}, true, cv::Scalar(0, 255, 0), 2);
```
Comment (maoyangchien) [inline src/OptrisColorconvert.cpp:220]: Why did you change from 1 to 2 ?

Comment (SFhmichael) [reply src/OptrisColorconvert.cpp:220]: 線粗一點分的群看比較清楚XD
